Core.DomainServices/Repositories/KLE/KLEStandardRepository.cs

```diff
@@ -255,6 +261,7 @@ private void RemoveTaskRef(IEnumerable<KLEChange> kleChanges)
                 update.Item1.OwnedByOrganizationUnitId = ownedByOrgnizationUnitId;
                 update.Item1.ActiveFrom = update.Item2.ActiveFrom;
                 update.Item1.ActiveTo = update.Item2.ActiveTo;
+                _domainEvents.Raise(new EntityUpdatedEvent<TaskRef>(update.Item1));
```

Comment:
spørgsmålet er om man skal vente med at raise alle events indtil transaktionen er committed. Ellers vil opdateringen i baggrunden jo ikke have effekt (kører jo i en anden tråd og derfor uden for context) :-)